README.md

```diff
@@ -112,6 +113,109 @@ plugins: [
 ];
 ```
 
+## Running Babel on the generated code
+
+By setting `options.transformGenerated: true`, you can configure rollup-plugin-babel to run Babel on the output files instead of the input files. This can be used to perform code transformations on the resulting chunks and is the only way to transform Rollup's auto-generated code. By default, the plugin will be applied to all outputs:
+
+```js
+// rollup.config.js
+import babel from 'rollup-plugin-babel';
+export default {
+	input: 'main.js',
+	plugins: [babel({ transformGenerated: true })],
+	output: [
+		{ file: 'bundle.cjs.js', format: 'cjs' },
+		{ file: 'bundle.esm.js', format: 'esm' },
+	],
+};
+```
+
+If you only want to apply it to specific outputs, you can use it as an output plugin:
+
+```js
+// rollup.config.js
+import babel from 'rollup-plugin-babel';
+export default {
+	input: 'main.js',
+	output: [
+		{ file: 'bundle.js', format: 'esm' },
+		{
+			file: 'bundle.min.js',
+			format: 'esm',
+			plugins: [babel({ transformGenerated: true })],
+		},
+	],
+};
+```
+
+The `include`, `exclude` and `extensions` options are ignored when the `transformGenerated` option is used and will produce warnings, and there are a few more points to note that users should be aware of.
+
+### Finding .babelrc files
```

Comment:
For it to be correct u would also have to mention babel.config.js. How we load config files etc should be described in a common section of the docs (rather than in transformGenerated subsection). I think we shouldnt mention specific config files here - just link to Babel docs where all possibilities are described. In here we could only mention that thing about stdout, javascript API etc. 

If this would be described in a common section then in here we should just mention that configs are loaded “for” output files - rather than for input ones.

Reply:
Thanks for pointing that out, I will try to improve this.